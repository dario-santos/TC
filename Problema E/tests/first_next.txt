4
S -> a
S -> b
A -> _
A -> S a
