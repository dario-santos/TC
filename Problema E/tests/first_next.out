NULL(S) = False
NULL(A) = True
FIRST(S) = a b
FIRST(A) = a b
FOLLOW(S) = # a
FOLLOW(A) =
